src/components/FilterInput.svelte

```diff
@@ -1,5 +1,5 @@
 <script>
-  import throttle from "just-throttle";
+  const throttle = require("just-throttle");
```

Comment:
With lodash we can just do:

```suggestion
  import { throttle } from "lodash";
```

... and it'll work